test/fixtures/test-runner/output/coverage_failure.js

```diff
@@ -0,0 +1,14 @@
+// Flags: --expose-internals --experimental-test-coverage
```

Comment:
Optional nits in this file:

- For the monkey patching, you could use the top level `mock()` function.
- Instead of using `__proto__.constructor`, you could export the `TestCoverage` class. It's already an internal file and you're using `--expose-internals` here. If you do it, just leave a comment that it's exported for testing purposes.